Backpack/Rating/Classes/BPKRating.m

```diff
@@ -212,7 +212,11 @@ - (void)setUpConstraints {
     self.verticalLayoutConstraints = @[
         self.textSpacingConstraintVertical,
         [self.textWrapper.centerXAnchor constraintEqualToAnchor:self.ratingBubble.centerXAnchor],
-        [self.trailingAnchor constraintGreaterThanOrEqualToAnchor:self.ratingBubble.trailingAnchor]
+        [self.trailingAnchor constraintGreaterThanOrEqualToAnchor:self.ratingBubble.trailingAnchor],
+
+        // If the text is less wide than the bubble make sure the view in total
+        // is still large enough not to clip the bubble.
+        [self.widthAnchor constraintGreaterThanOrEqualToAnchor:self.ratingBubble.widthAnchor multiplier:1.0],
```

Comment:
Do we need to specify the `multiplier` if it's `1.0`?

Reply:
Nope, good point